controller/HandShake.php

```diff
@@ -20,14 +20,48 @@
 
 namespace oat\taoSync\controller;
 
+use oat\oatbox\user\LoginFailedException;
+use oat\tao\helpers\RestExceptionHandler;
 use oat\taoSync\model\server\HandShakeServerService;
 
-class HandShake extends \tao_actions_RestController
+class HandShake extends \tao_actions_CommonModule
 {
     const USER_IDENTIFIER = 'login';
 
+    private $responseEncoding = "application/json";
```

Comment:
A lot of duplication code from tao_actions_RestController, some trait can help with this.